From e8d3388beb7b3729b8f6174b773889c97ccd219b Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:23:34 +0100
Subject: [PATCH 2/3] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into
 lineage-16.0

commit 46d1954c86bab445eb1122323d397e7b04159141
Merge: 40ba52b9 08ef4924
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Mon Aug 13 03:14:50 2018 +0000

    Snap for 4949382 from 08ef492420df9fe9043137a18afd000f5ad9d2de to pi-qpr1-release

    Change-Id: Ib99c00f4a3bd9a6b079242ce32a088eb7c56a09b

commit 08ef492420df9fe9043137a18afd000f5ad9d2de
Merge: 1a15874b c3bc8bf5
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 17:29:19 2018 -0700

    [automerger skipped] Merge commit '6d489a5f' into aug10d-oc-dev am: 1c7dce84d8 am: fa03ea96b7
    am: c3bc8bf5be  -s ours

    Change-Id: I92ce0db6099b32f7a4cd3a84591f103f4ac4af4b

commit c3bc8bf5be369bb2dc3b6aa726f0e2637cdf5c80
Merge: 9c2867c8 fa03ea96
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 17:13:10 2018 -0700

    Merge commit '6d489a5f' into aug10d-oc-dev am: 1c7dce84d8
    am: fa03ea96b7

    Change-Id: I2026742053efc2cc85e8aead2c96fa5db3199719

commit fa03ea96b72ae80207a12b0f7089a1e5d7006dad
Merge: 32a92d0a 1c7dce84
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 16:24:22 2018 -0700

    Merge commit '6d489a5f' into aug10d-oc-dev
    am: 1c7dce84d8

    Change-Id: I634f146b629a2fbcf293619337683b59afd4b41a

commit 1a15874b8d79942bc9159bbf9dd72661256ba6b0
Merge: 3cb737ce 64b55ea8
Author: Jeff Sharkey <jsharkey@google.com>
Date:   Fri Aug 10 21:55:56 2018 +0000

    Merge "DO NOT MERGE. All untrusted selections must go through builder." into pi-dev

commit 1c7dce84d8c09b8991289a922f5d8c29f078f20d
Merge: 84cefc51 6d489a5f
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 15:50:34 2018 -0600

    Merge commit '6d489a5f' into aug10d-oc-dev

    Bug: 111085900
    Change-Id: I2f224f04d1ae39a64a9c0784fe0c9cab284d88e8

commit 6d489a5f574013d22fe16af00464cc1e9eaf0616
Merge: b2db30a2 b760a3bb
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 11:43:47 2018 -0600

    Merge commit 'b760a3bb' into aug10d-nyc-mr2-dev

    Bug: 111085900
    Change-Id: Ie12be47f760402566801eb42f5defa0611a51ca2

commit b760a3bb46b86a820303cb11bb506cd01c777a4d
Merge: 3439b74e 21428513
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 11:42:45 2018 -0600

    Merge commit '21428513' into aug10d-cw-f-dev

    Bug: 111085900
    Change-Id: Ia1b146d58b74988ba962bc8e5155d027434f7e0e

commit 21428513cbbf61b7d3c058fcb95736dc0848127d
Merge: 7f0b5c44 25e04e39
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 11:38:26 2018 -0600

    Merge commit '25e04e39' into aug10d-nyc-mr1-dev

    Bug: 111085900
    Change-Id: I01cee751563a32e0b1deb181f820339d21ed6272

commit 25e04e397876bf938d0e5ab86e64f6e447ba7ba6
Merge: 52b9a670 d728a508
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Fri Aug 10 11:36:49 2018 -0600

    Merge commit 'd728a508' into aug10d-nyc-dr1-dev

    Bug: 111085900
    Change-Id: Idc5260b874063b7a5c7fe07b95883467ed8be449

commit d728a508d651060676f93b252f5c62927a3728a1
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 16 13:18:04 2018 -0600

    DO NOT MERGE. All untrusted selections must go through builder.

    When accepting untrusted selections, they must be passed directly
    to SQLiteQueryBuilder to ensure that setStrict() can be applied to
    check for malicious callers sending unbalanced parentheses.  This
    means we can't mix local and remote selections; they always need to
    be kept separate.

    Use newly added SQLiteQueryBuilder functionality to apply strict
    detection to update() and delete() calls.

    Only allow the owner of a particular download to query the headers
    for that download.  Only delete headers for a download once we've
    confirmed that caller can modify that download.

    Test: atest packages/providers/DownloadProvider/tests/
    Test: atest cts/tests/app/src/android/app/cts/DownloadManagerTest.java
    Bug: 111085900
    Change-Id: I9fd8e0d3cf80d7603bf0092f36fe449467090821
    Merged-In: I9fd8e0d3cf80d7603bf0092f36fe449467090821

commit 40ba52b93b86ebe6c6336002df4d9ba14139eb3f
Merge: 10cde684 3cb737ce
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Mon Aug 6 03:12:26 2018 +0000

    Snap for 4933870 from 3cb737cefc1e296e75d323b2d221ac8ceaafa6b8 to pi-qpr1-release

    Change-Id: I6d48f375f3847c84a9405ccdd7d2951901d0be7e

commit 3cb737cefc1e296e75d323b2d221ac8ceaafa6b8
Author: Bill Yi <byi@google.com>
Date:   Fri Aug 3 17:11:20 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ie16a2d3550ec50409c2396f439222fdec04af5e7

commit 64b55ea82b1f394369237601ae1f1c78b776aabc
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 16 13:18:04 2018 -0600

    DO NOT MERGE. All untrusted selections must go through builder.

    When accepting untrusted selections, they must be passed directly
    to SQLiteQueryBuilder to ensure that setStrict() can be applied to
    check for malicious callers sending unbalanced parentheses.  This
    means we can't mix local and remote selections; they always need to
    be kept separate.

    Use newly added SQLiteQueryBuilder functionality to apply strict
    detection to update() and delete() calls.

    Only allow the owner of a particular download to query the headers
    for that download.  Only delete headers for a download once we've
    confirmed that caller can modify that download.

    Test: atest packages/providers/DownloadProvider/tests/
    Test: atest cts/tests/app/src/android/app/cts/DownloadManagerTest.java
    Bug: 111085900
    Change-Id: I9fd8e0d3cf80d7603bf0092f36fe449467090821
    Merged-In: I9fd8e0d3cf80d7603bf0092f36fe449467090821

commit 10cde684808f7b6c68dffd0f2f48cce8b50ec0ca
Merge: ce9f204a 9c2867c8
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 20:57:48 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e am: 84cefc516e am: 32a92d0a73
    am: 9c2867c8a1

    Change-Id: I93ec2760b7db41dcb551bcdf16c6d38df078ab52

commit 9c2867c8a12ad9bc04a1d8bdd4fc3879f3ffc178
Merge: b627002e 32a92d0a
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 19:05:17 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e am: 84cefc516e
    am: 32a92d0a73

    Change-Id: Ib38b3c09bfaf7e159712b09b6c8f1b82ed33833d

commit 32a92d0a737d42af54f44b9f1128fa7ea676f13a
Merge: b2de5829 84cefc51
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 17:30:20 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e
    am: 84cefc516e

    Change-Id: Iabab984d78bad2ca6b5904e31482044e4627d02a

commit ce9f204ac493f000cd3020e195fd5038d0cec1e2
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 9 12:16:26 2018 -0600

    Remove "public" download feature.

    It was never a supported API, and has been reported as causing
    security issues, so remove it.

    Bug: 111084083
    Test: builds
    Change-Id: I26345b192ffd55216bb8c8fdb82cb5869d68d3db

commit 84cefc516ed8691b11670b1cfd80216969cfbfef
Merge: d08c19e2 b2db30a2
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:03 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e

    Change-Id: I4e2fda8c302ddfaa0edeb226029c94ff3771f89f

commit b2db30a21eaac0ebc033bc5a824b146633bedda0
Merge: 3a1bec71 3439b74e
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:02 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a

    Change-Id: I1509a4dc66f4d57701d54b7743de3bd9915258c7

commit 3439b74e4af9c7303310545736578d1fcf518412
Merge: 09fb61e7 7f0b5c44
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:01 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8

    Change-Id: I7b24ea92c1227e4581e72b551e3ebac4b4bd3528

commit 7f0b5c44c87b156d050d8183603ed2d828e0a874
Merge: 79f65e6e 52b9a670
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:17:59 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8

    Change-Id: I55c96981db12008e619333f67ea8033e79ca5bd2

commit 52b9a670b80f5843316c4e076a40abd178e28b9c
Merge: bf2ee732 35e12311
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:17:58 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b

    Change-Id: I0347a0a15e99a224df13b756dc3d27de23835cae

commit 35e123117be9ec5d61dbaea60f6eac06c0e80dc4
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 9 12:16:26 2018 -0600

    Remove "public" download feature.

    It was never a supported API, and has been reported as causing
    security issues, so remove it.

    Bug: 111084083
    Test: builds
    Change-Id: I26345b192ffd55216bb8c8fdb82cb5869d68d3db

commit aeda68c64c46fd3ea28bbaa78d5c4f89b49eadd9
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 09:04:59 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ibe041341d032a0eb9b0f31b74db37c2e3de38df9

Change-Id: Id60cd196b43cd9b673e675c33cac5d6224f644cb
---
 res/values-fr/strings.xml | 2 +-
 res/values-hy/strings.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index ed341cc5..83489be8 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_label" msgid="3658948994665187911">"Gest. de tél."</string>
+    <string name="app_label" msgid="3658948994665187911">"Gestionnaire de téléchargement"</string>
     <string name="storage_description" msgid="7982444311558023664">"Téléchargements"</string>
     <string name="permlab_downloadManager" msgid="7779544811202855500">"Accéder au gestionnaire de téléchargement."</string>
     <string name="permdesc_downloadManager" msgid="4237406545998908947">"Permet à l\'application d\'accéder au gestionnaire de téléchargement et de l\'utiliser pour télécharger des fichiers. Les applications malveillantes peuvent utiliser cette fonctionnalité pour perturber les téléchargements et accéder à des informations confidentielles."</string>
diff --git a/res/values-hy/strings.xml b/res/values-hy/strings.xml
index a1a48e93..506158b1 100644
--- a/res/values-hy/strings.xml
+++ b/res/values-hy/strings.xml
@@ -42,7 +42,7 @@
     <string name="wifi_required_body" msgid="3067694630143784449">"Դուք պետք է օգտագործեք Wi-Fi՝ այս <xliff:g id="SIZE">%s </xliff:g> ներբեռնումն ավարտելու համար: \n\nՀպեք <xliff:g id="QUEUE_TEXT">%s </xliff:g>՝ սկսելու այս ներբեռնումը հաջորդ անգամ, երբ միացած կլինեք որևէ Wi-Fi ցանցի:"</string>
     <string name="wifi_recommended_title" msgid="7441589306734687400">"Հերթագրե՞լ ավելի ուշ ներբեռնելու համար:"</string>
     <string name="wifi_recommended_body" msgid="1314735166699936073">"Այս <xliff:g id="SIZE">%s </xliff:g> ներբեռնումը հիմա սկսելը կարող է կարճեցնել ձեր մարտկոցի կյանքը և/կամ բերել ձեր բբջային տվյալների կապի ավելորդ օգտագործման, որը կարող է ուղեկցվել ծախսերով ձեր բջջային օպերատորի կողմից՝ կախված ձեր տվյալների փաթեթից:\n\n Հպեք <xliff:g id="QUEUE_TEXT">%s</xliff:g> ստորև` այս ներբեռնումը սկսելու հաջորդ անգամ, երբ միացած կլինեք որևէ Wi-Fi ցանցի:"</string>
-    <string name="button_queue_for_wifi" msgid="422576726189179221">"Հերթականություն"</string>
+    <string name="button_queue_for_wifi" msgid="422576726189179221">"Հերթագրել"</string>
     <string name="button_cancel_download" msgid="2430166148737975604">"Չեղարկել"</string>
     <string name="button_start_now" msgid="792123674007840864">"Սկսել հիմա"</string>
     <plurals name="notif_summary_active" formatted="false" msgid="7290448463204837173">
-- 
2.17.1

